scripts/rollup/bundles.js

```diff
@@ -396,6 +396,15 @@ const bundles = [
     externals: ['jest-diff'],
   },
 
+  /******* Jest Scheduler (experimental) *******/
+  {
+    bundleTypes: [NODE_DEV, NODE_PROD, FB_WWW_DEV, FB_WWW_PROD],
+    moduleType: ISOMORPHIC,
+    entry: 'jest-mock-scheduler',
+    global: 'JestMockScheduler',
+    externals: ['jest-diff'],
```

Comment:
I assumed this was required for some reason, since it was listed as a dependency for the `jest-react` package (even though that package didn't reference it anywhere that I saw).

Reply:
Probably copypasta too.

Reply:
Oh well. We can rip it out in of both then.

Reply:
#14372